bootstrap.py

```diff
@@ -1,112 +1,287 @@
 #!/usr/bin/env python3
 
 import argparse
+import collections
 import os
-import shutil
+import stat
 import sys
 import tempfile
 import textwrap
-
-import constants
 import utils
+import yaml
+
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+
+excluded_files = collections.defaultdict(set)
+
+
+class AppendExclude(argparse.Action):
```

Comment:
This will be removed, it was one option for implementation, but not worth doing because we will switch to yaml config instead of CLI args

Reply:
However, this could be useful for the update command, to skip certain files for a particular run.